pipeline/metadata/6.0.yaml

```diff
@@ -229,6 +229,22 @@ suites:
       - rgw
       - stage-1
 
+  - name: "test-rgw-sse-s3-encryption-GA"
+    suite: "suites/pacific/rgw/tier-2_sse_s3_encryption.yaml"
+    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
```

Comment:
@viduship it looks like path is incorrect here and we might not able to list this suite in custom jenkins run while executing it for 6.x build